Library/Homebrew/extend/ENV/std.rb

```diff
@@ -287,7 +287,7 @@ def cxx11
     if compiler == :clang
       append "CXX", "-std=c++11"
       append "CXX", "-stdlib=libc++"
-    elsif compiler =~ /gcc-(4\.(8|9)|5)/
+    elsif compiler =~ /gcc-(4\.(8|9)|5|6)/
```

Comment:
> May even be worth having a method for this regex in ENV/shared.rb as they are the same in both places.

Maybe even a proper version check (`>= 4.8`) so we don't need to fix this with the next release. It's unlikely GCC is going to drop C++11 support in a future release. Something like this maybe?
```ruby
elsif compiler =~ /gcc-(\d+(?:\.\d)?)/ && Version.new($1) >= Version.new("4.8")
  append "CXX", "-std=c++11"
```

(I'm not super happy with the use of `$1` here, as that looks way too magical.)